app/controllers/admin/screened_ip_addresses_controller.rb

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 
+require_dependency 'ip_addr'
```

Comment:
Just curious, why do we need to add this line in the context of this PR?

Reply:
This controller uses IPAddr class and it was loaded from Ruby's stdlib without loading the monkey patches from lib/ip_addr.rb.

Reply:
Icic. I find this confusing and followed up with a fix in another [PR](https://github.com/discourse/discourse/pull/15516)